css/components/list.css

```diff
@@ -0,0 +1,63 @@
+/* === css for List === */
+
+/* === css for normal list === */
+.list{
+    box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.6);
+    padding: .9rem  1.2rem;
+}
+.list-text{
+    font-size: var(--h3-font-size);
+    border-bottom: .1rem solid var(--text-color-light);
+    margin-bottom: var(--mb-1);
+    color: var(--primary);
```

Comment:
You can be more consistent with your naming. color name : `--text-color-light` and `primary` doesn't have consistency.

Reply:
sure!!! done with the suggestion